python/lsst/ap/association/diaForcedSource.py

```diff
@@ -108,11 +110,17 @@ def run(self,
             Used to assure that the pipeline includes all diaObjects that were
             updated in case one falls on the edge of the CCD.
         expIdBits : `int`
-            Bit length of the exposure id.
+            Bit length of the exposure id.  Deprecated in favor of
```

Comment:
Is there a ticket number for that removal that you could mention in this docstring and above the `if` below?